### ISSUE TITLE
Dart Sass 2.0.0 compatibility

### ISSUE THREAD

**Issue by loebi-ch** [jonsuh/hamburgers#107]
Hello

I get the following DEPRECATION WARNING:

```
DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($hamburger-layer-height, 2)

More info and automated migrator: https://sass-lang.com/d/slash-div

  ╷
7 │       top: $hamburger-layer-height / 2;
  │            ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ╵
    node_modules\hamburgers\_sass\hamburgers\types\_elastic.scss 7:12  @import
    node_modules\hamburgers\_sass\hamburgers\hamburgers.scss 87:9      @import
    Client\app\components\menu\menu.component.scss 4:9                 root stylesheet
```

The error is self-explanatory, respectively you can read more here: [https://sass-lang.com/d/slash-div](https://sass-lang.com/d/slash-div)

**Comment by nstungcom** [comment]
@loebi-ch There is already a PR for this issue but not merged yet, see: [PR105](https://github.com/jonsuh/hamburgers/pull/105) Hopefully @jonsuh or any contributors can merge the PR soon.

**Comment by mtuzinskiy** [comment]
@loebi-ch I'd like to point out that it's not only about Dart, it's a general SASS compatibility, that makes sense for any non-Dart framework. Could you please change the title of the issue?  I hope that it'll make it more visible for the contributors @jonsuh

**Comment by ianhobbs** [comment]
+1 for this

**Comment by verzola** [comment]
+1 for this, the warnings are making the logs very messy:
![image](https://user-images.githubusercontent.com/2985522/148533904-27d45702-fdfe-4317-b5ab-984e6dff8570.png)

**Comment by rhcarlosweb** [comment]
+1 for this

**Comment by Maxxa** [comment]
+1

**Comment by oguilleux** [comment]
Please @jonsuh can you take a moment and look at this. Thank you !

**Comment by mtuzinskiy** [comment]
I think it's time for us to make a fork guys... Obviously author doesn't want to continue the project

**Comment by jonsuh** [comment]
This issue should be fixed via https://github.com/jonsuh/hamburgers/pull/105

**Comment by oguilleux** [comment]
Thanks a lot @jonsuh ! Will npmjs automatically update to latest version ? https://www.npmjs.com/package/hamburgers - I'd like to send to my different projects.

**Comment by jonsuh** [comment]
@oguilleux Yes, I just cut a release which was also published on npm. It should be updated https://www.npmjs.com/package/hamburgers

**Comment by julkue** [comment]
While this change was with good intend, it should have been a major version. The release broke a legacy application of mine, since it used `^1.1.3`  (since 2018).
